data/transactions/application_test.go

```diff
@@ -26,17 +25,6 @@ import (
 	"github.com/algorand/go-algorand/data/basics"
 )
 
-func TestApplicationCallFieldsNotChanged(t *testing.T) {
```

Comment:
Something weird is going on. Travis just failed on this test even though it's missing from this PR:
https://travis-ci.com/github/algorand/go-algorand/jobs/366736430

Reply:
I removed it because it was passing locally and failing on Travis, which confused me since it should be completely deterministic. The fact that it failed on Travis despite the test being completely gone indicates that something is broken.

Reply:
Ah, ok, figured it out. It was a bad merge -- the test moved over to the `apply` package after the apply methods refactor, so there were two copies of this unit test. Added it back here and removed it from apply